src/libsyntax/feature_gate.rs

```diff
@@ -396,6 +396,9 @@ declare_features! (
     // Infer outlives requirements; RFC 2093
     (active, infer_outlives_requirements, "1.26.0", Some(44493), None),
 
+    // Infer outlives requirements; RFC 2093
```

Comment:
This should use a different comment than the previous entry, to distinguish it. `Infer 'static outlives requirements`, perhaps?